### PR TITLE
Add user profile page

### PR DESCRIPTION
**Pull request by adrian-gomez**

### CODE REVIEW

File: src/Page/User/User.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    return linkClassNames;
+  };
+
+  return (
+    <div>
+      <Row>
+        <Col>
+          <nav className="nav nav-tabs">
+            <a href="/user/profile" className={linkClass('/user/profile')}>
```
Comment (agustinrhcp) [inline src/Page/User/User.js]: don't use `a` for internal links, but `ReactRouterDorm.Link`. Also, there's a NavLink, which accepts an `acctiveClass` which is added to the Dom based on the url (:

Comment (adrian-gomez) [reply src/Page/User/User.js]: thanks!

File: src/Component/Header/Header.js [hunk 2]
```diff
@@ -158,7 +158,14 @@ const Header = withRouter(({ history }) => {
                 </NavDropdown.Item>
               </NavDropdown>
 
-              <NavDropdown title={user.name} id="header-nav-profile">
+              <NavDropdown
+                title={
+                  <>
```
Comment (agustinrhcp) [inline src/Component/Header/Header.js]: you don't need the `<> </>`

File: src/Component/PhonesFormInput/PhoneFormInput.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+};
+
+const PhoneFormInput = ({ index, phone, onChange, onRemove }) => {
+  const localPhone = { ...defaultPhone, ...phone };
+
+  const handlePhoneRemove = () => {
+    onRemove(index);
+  };
+
+  const handleChange = ({ target }) => {
```
Comment (agustinrhcp) [inline src/Component/PhonesFormInput/PhoneFormInput.js]: I'm refactoring this as soon as you merge it.

File: src/Component/PhonesFormInput/PhonesFormInput.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+import PhoneFormInput from './PhoneFormInput';
+
+const PhonesFormInput = ({
+  phones,
+  onPhoneAdd,
+  onPhoneUpdated,
+  onPhoneRemove,
+}) => {
+  const localPhones = (phones || []).map(phone => {
+    const phoneWithId = phone;
```
Comment (agustinrhcp) [inline src/Component/PhonesFormInput/PhonesFormInput.js]: `map(() => { id: short_id.generate(), ...phone }` or you can use `default` or something like that from 
loadash

however, isn't this refreshed every time the component rerenders?

File: src/Page/User/Profile/Profile.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
+import React, { useContext } from 'react';
+import { Card } from 'react-bootstrap';
+import _ from 'lodash';
+import { toast } from 'react-toastify';
+
+import { GlobalStateDispatchContext } from '../../../Component/GlobalStateProvider';
+import useGlobalState from '../../../Hook/useGlobalState';
+import { update } from '../../../Service/User';
```
Comment (agustinrhcp) [inline src/Page/User/Profile/Profile.js]: I'm fixing this long imports later.

File: src/Page/User/Profile/Profile.js [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+const UserProfile = () => {
+  const [{ user }] = useGlobalState();
+  const globalDispatch = useContext(GlobalStateDispatchContext);
+
+  const handleSubmit = params => {
+    const { firstName, lastName, termsAccepted, baaAccepted } = params;
+    const userParams = _.pick(params, ['email', 'phones']);
+
+    userParams.first_name = firstName;
```
Comment (agustinrhcp) [inline src/Page/User/Profile/Profile.js]: if only Agustin had some code to make everything snake_case or camelCase 🤷‍♂

File: src/Page/User/Profile/UserForm.js [hunk 7]
```diff
@@ -0,0 +1,328 @@
...
+        </Form.Group>
+
+        <PhonesFormInput
+          phones={phones}
+          onPhoneAdd={handlePhoneAdd}
+          onPhoneRemove={handlePhoneRemove}
+          onPhoneUpdated={handlePhoneUpdate}
+        />
+
+        <Col md={{ span: 8, offset: 4 }}>
```
Comment (agustinrhcp) [inline src/Page/User/Profile/UserForm.js]: Feel free to move html blocks into dummy components if they take too much space

File: webpack.config.common.js [hunk 8]
```diff
@@ -28,7 +28,11 @@ const config = {
         exclude: [/node_modules/, /\.test.js$/],
       },
       {
-        test: /\.scss$/,
+        test: /\.css$/i,
```
Comment (agustinrhcp) [inline webpack.config.common.js]: `s?css` ?

Comment (adrian-gomez) [reply webpack.config.common.js]: thanks!

File: src/Component/PhonesFormInput/PhonesFormInput.js (hunk 4 above)
Comment (adrian-gomez) [reply src/Component/PhonesFormInput/PhonesFormInput.js]: thanks! &&
it is, this was the only way this worked 🤷‍♂

**Pull request merged**